app/lib/features/chat/widgets/text_message_builder.dart

```diff
@@ -143,6 +144,11 @@ class _TextWidget extends ConsumerWidget {
               maxLines: isReply ? 3 : null,
             )
           : Html(
+              onLinkTap: (url) async {
```

Comment:
We have a `openLink` in `app/lib/common/utils/utils.dart` that tries to figure out whether something is an app internal link (e.g. to a Task-item) or something we try to open with the external launcher. Please use that helper here.

Reply:
Sure, let me do changes accordingly